internal/service/accesslistapikey/resource_access_list_api_key_migration_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
 )
 
-func TestAccMigrationProjectAccesslistAPIKey_SettingIPAddress(t *testing.T) {
+func TestMigProjectAccesslistAPIKey_SettingIPAddress(t *testing.T) {
```

Comment:
> TestAccMigrationXXX migration test names are renamed to TestMigXXX so acc test can be run only with a regexp and they don't need the skip param. Also we make test go "cacheable-friendly" as go test with skip param can't use cache.

[not here - food for thoughts] Maybe it is worth thinking of ways to make sure new tests are following the correct name format

Reply:
that's fine, although now the only requirement is that test names start with TestAcc, migration tests with TestMig, and unit tests with Test and not TestAcc nor TestMig.